tests/components/plex/test_config_flow.py

```diff
@@ -367,13 +373,11 @@ async def test_option_flow(hass):
 
     with patch("plexapi.server.PlexServer", return_value=mock_plex_server), patch(
         "plexapi.myplex.MyPlexAccount", return_value=MockPlexAccount()
-    ), patch("homeassistant.components.plex.PlexWebsocket.listen") as mock_listen:
+    ), patch("homeassistant.components.plex.PlexWebsocket", return_value=MockWebsocket):
```

Comment:
I don't think we need to specify a return value. The default should be ok.
```suggestion
    ), patch("homeassistant.components.plex.PlexWebsocket"):
```

Reply:
Without some spec provided the patching fails as one of the class methods is a coroutine:
```
TypeError: a coroutine was expected, got <MagicMock name='PlexWebsocket().listen()' id='140007161242512'>.
```
Setting `autospec=True` seems to handle this fine, however.